<launch>

  <node pkg="pose_estimate" type="pose_estimate" name="pose_estimate" output="screen">
 	
	<param name="sub_imu_topic" value="/imu0" type="str" />
	<param name="sub_mag_topic" value="/mag" type="str" />
	<param name="sub_gps_topic" value="/gps" type="str" />
	<param name="hz" value="100" type="int" />
	<param name="use_mag" value="false" type="bool" />
	<!-- EKF Mahony Madgwick /--> 
	<param name="algorithm" value="EKF" type="str" /> 
  </node>


</launch>
